src/system/documents/item.ts

```diff
@@ -518,9 +518,14 @@ export class CosmereItem<
         );
 
         // Make the graze pool and roll it
+        const defaultGraze = roll.terms.every(
+            (term) => !(term instanceof foundry.dice.terms.DiceTerm),
+        )
```

Comment:
Minor nitpick: it's faster to do `!roll.terms.some(term => foundry.dice.terms.DiceTerm)` because that will exit out early if a single DiceTerm is found, whereas every will always run the full iteration